### PR TITLE
Load mentees in the manage mentees view

### LINKED ISSUE #39: Bind the API with the manage mentees view
**Is your feature request related to a problem? Please describe.**
- We need to bind the manage mentee view with the relevant API

**Describe the solution you'd like**
- Bind the API with the manage mentees view

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #39 

## Goals

- Wiring the backend with the frontend

## Approach

- Wired the manage mentees view with the API
##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/scenes/Dashboard/scenes/ManageMentees/index.tsx [hunk 1]
```diff
@@ -1,55 +1,75 @@
...
+                </Button>,
+                <Button key="more" type="default">
+                  More
+                </Button>,
+              ]}
+            >
+              <List.Item.Meta
+                avatar={<Avatar src={item.profile.imgUrl} />}
+                title={
+                  <a href="#">
```
Comment (jayasanka-sack) [inline src/scenes/Dashboard/scenes/ManageMentees/index.tsx]: We can add the url here. make it to open in another tab

Comment (jayasanka-sack) [reply src/scenes/Dashboard/scenes/ManageMentees/index.tsx]: That's all, reset looks good for me

Comment (anjula-sack) [reply src/scenes/Dashboard/scenes/ManageMentees/index.tsx]: What about the API endpoint?

**Review by jayasanka-sack — APPROVED** [review]
Thanks @anjula-sack !

**Pull request merged**